mobile/src/libs/types/navigation/chat-navigation-parameter-list.type.ts

```diff
@@ -0,0 +1,8 @@
+import { type ChatScreenName } from '#libs/enums/navigation/chat-screen-name.enum';
```

Comment:
Import from `libs/enums/enums.ts`, where all enums should be reexported

Reply:
At now you are importing it from `libs/enums/navigation/navigation`, but not `libs/enums/enums`)

Reply:
Thank you, you are right. I've already changed it